transformations/tests/conftest.py

```diff
@@ -68,3 +72,45 @@ def my_test(frontend):
             params += [f]
 
     return params
+
+
+def write_env_launch_script(here, binary, args):
+    # Write a script to source env.sh and launch the binary
+    script = Path(here/f'build/run_{binary}.sh')
+    script.write_text(f"""
+#!/bin/bash
+
+source env.sh >&2
+bin/{binary} {' '.join(args)}
+exit $?
+    """.strip())
+    script.chmod(0o750)
+
+    return script
+
+
+def local_loki_setup(here):
+    lokidir = Path(__file__).parent.parent.parent
+    target = here/'source/loki'
+    backup = here/'source/loki.bak'
+    bundlefile = here/'bundle.yml'
+
+    # Do not overwrite any existing Loki copy
+    if target.exists():
+        if backup.exists():
+            shutil.rmtree(backup)
+        shutil.move(target, backup)
+
+    # Change bundle to symlink for Loki
+    bundle = yaml.safe_load(bundlefile.read_text())
```

Comment:
Is this still needed now? I thought by calling the create step with a pointer to the loki dir we could cut out the need to modify bundle.yml entirely?

Reply:
I wanted to keep the assert where we check that loki is in the original `bundle.yml`. If we don't want to check that then yeah we can get rid of this completely.